cmd/cert-checker/main_test.go

```diff
@@ -422,6 +427,31 @@ func TestGetCertsEmptyResults(t *testing.T) {
 	test.AssertNotError(t, err, "Failed to retrieve certificates")
 }
 
+// create emptyDB for testing that null response from mariadb are handled properly
+type emptyDB struct{}
+
+// set SelectNullInt to respond with false to reflect the empty DB response
+func (db emptyDB) SelectNullInt(_ string, _ ...interface{}) (sql.NullInt64, error) {
+	return sql.NullInt64{Valid: false},
+		nil
+}
+
+// define Select so the compiler is happy. Not used.
+func (db emptyDB) Select(output interface{}, _ string, _ ...interface{}) ([]interface{}, error) {
+	return nil, nil
+}
+
+// check that a NULL response from the DB will be handled correctly
+func TestGetCertsNullResults(t *testing.T) {
+	saDbMap, err := sa.NewDbMap(vars.DBConnSA, sa.DbSettings{})
+	test.AssertNotError(t, err, "Couldn't connect to database")
+	checker := newChecker(saDbMap, clock.NewFake(), pa, kp, time.Hour, testValidityDurations)
+	checker.dbMap = emptyDB{}
+
+	err = checker.getCerts(false)
+	test.AssertError(t, err, "Failed to handle NULL DB response")
```

Comment:
just a phrasing nit:
```suggestion
	test.AssertError(t, err, "Should have gotten error from empty DB")
```